cli/src/main.rs

```diff
@@ -79,7 +80,11 @@ fn main() {
                         num_clusters: num_tables,
                     },
                 };
-                encode_stream(&mut in_file, &mut out_file, threads, encoding_strategy);
+                let threads_val = match threads {
```

Comment:
Did you intentionally not use `unwrap_or` here (e.g. for the sake of readability)?

Reply:
Not intentional. Thank you for the suggestion. [Applied unwrap_or](https://github.com/torfmaster/ribzip2/pull/15/files#diff-37970e7d817d4316a093bb967d74ff7e26d9c6c01d93f5a79d3d195ba9f6155bR83)